fuel/transformers/__init__.py

```diff
@@ -60,6 +61,24 @@ def get_epoch_iterator(self, **kwargs):
         self.child_epoch_iterator = self.data_stream.get_epoch_iterator()
         return super(Transformer, self).get_epoch_iterator(**kwargs)
 
+    def get_data(self, request=None):
+        if self.batch_input:
+            return self.get_data_from_batch(request)
+        else:
+            return self.get_data_from_example(request)
+
+    def get_data_from_example(self, request=None):
```

Comment:
It is tempting to propose `transform_batch` and `transform_example` that look nice with the `Transformer` name, but those do not reflect that multiple input entries can make one output entry and vice versa.

I do not mind merging it as it to keep workflow fast, but I will create an issue because such names do not really invite to quickly write your own `Transformer`.